src/core/ir/evaluation_context.rs

```diff
@@ -23,9 +24,11 @@ pub struct EvaluationContext<'a, Ctx: ResolverContextLike<'a>> {
 
     // Overridden Arguments for Async GraphQL Context
     graphql_ctx_args: Option<Arc<Value>>,
+
+    pub type_name: Option<TypeName>,
```

Comment:
Add some doc about what is being stored here.

Reply:
added doc